src/ctapipe/image/tests/test_extractor.py

```diff
@@ -603,7 +678,7 @@ def test_waveform_extractor_factory_args(subarray):
 
 def test_extractor_tel_param(toymodel):
     waveforms, subarray, _, _, _, _ = toymodel
-    _, n_samples = waveforms.shape
+    n_samples = waveforms.shape[-1]
```

Comment:
Prefer the form `_, _, n_samples =` for consistency with L190, L693, and L714 (among others).

Reply:
Why unpack the tuple if you only need just one element?